handbook/communication/team_chat.md

```diff
@@ -9,7 +9,7 @@ If you are a new member of the engineering team, you'll most likely want to join
 - #general - Anything about Sourcegraph in general (that doesn't fit a more specific channel below).
 - #random - Cool things you're doing in life, cat photos, jokes, vacation photos, etc.
 - #progress - Share cool/interesting progress with the team
-- #thanks - Appreciate someone's work? You can thank them here! :)
+- #thanks - Appreciate someone's work? You can thank them here! :) {#thanks}
```

Comment:
What is the purpose of this edit?

Reply:
It adds an inline page anchor so you can link to `https://about.sourcegraph.com/handbook/communication/team_chat#thanks`.